src/components/popups/ProfileSettings.tsx

```diff
@@ -137,6 +140,7 @@ const ProfileSettings = ({ isOpen, onClose, onProfileOpen }) => {
         setRepPassword("");
         onClose();
         onProfileOpen(true);
+        setIsLoading(false);
```

Comment:
same goes here. what if request fails, then directly in catch block, and things are never reset

Reply:
Moved down